CHANGELOG.md

```diff
@@ -0,0 +1,66 @@
+# Changes Since v2.6.0
```

Comment:
Above this line should be the explanation for what the CHANGELOG.md is as it appears in 2.x.  Maybe also a new bit about how this changelog is just for the 3.x series.

This line should read
```
## [v3.0.0](with link to release)
```